modules/smart-agent_redis/variables.tf

```diff
@@ -1,5 +1,11 @@
 # Module specific
 
+variable "from_otel" {
+  description = "Set to true when metrics are sent by otel collector instead of smart-agent"
+  type        = bool
+  default     = false
```

Comment:
isn't otel now the new default? to set to true by default?
(or maybe reverse the variable meaning: "use legacy" ?)

Reply:
given that the module is named `smart-agent_redis`, I would keep the default behavior to be smart-agent (+ it avoids a breaking change)

Reply:
yes please keep false